docs/yaml/elements/street_segment.yaml

```diff
@@ -112,6 +125,26 @@ tags:
     **StartHouseNumber**, the implementation should ignore the element containing
     it.
   type: xs:integer
+- _name: HouseNumberPrefix
+  csv-header-name: house_number_prefix
+  csv-type: xs:string
+  description: Part of a street address. It may contain letters or slashes (e.g.,
+    'B' in 'B22 Main St'). If this value is present then **StartHouseNumber** must
+    be equal to **EndHouseNumber**. This field cannot be used if **IncludesAllAddresses**
+    or **IncludesAllStreets** are true.
+  error: If the field is not present of invalid, the implementation is required to
+    ignore it.
+  type: xs:string
+- _name: HouseNumberSuffix
+  csv-header-name: house_number_suffix
+  csv-type: xs:string
+  description: Part of a street address. It may contain letters or slashes (e.g.,
+    1/2 in '22 1/2 Main St'). If this value is present then **StartHouseNumber** must
+    be equal to **EndHouseNumber**. This field cannot be used if **IncludesAllAddresses**
+    or **IncludesAllStreets** are true.
+  error: If the field is not present of invalid, the implementation is required to
+    ignore it.
```

Comment:
Slight grammar issues on both `house_number_prefix` and `house_number_suffix` in the Error Handling fields: "If the field is not present of invalid..." should be "If the field is not present **or** invalid..."

Reply:
Thank you! This has been fixed.